packages/frontend2/src/app/(legacy)/scaling-next/projects/[slug]/tvl-breakdown/_components/tvl-breakdown-page-header.tsx

```diff
@@ -0,0 +1,43 @@
+import { type UnixTime } from '@l2beat/shared-pure'
+import Image from 'next/image'
+import { formatTimestampToDateWithHour } from '~/utils/dates'
+
+interface Props {
+  title: string
+  slug: string
+  tvlBreakdownDate: UnixTime
+}
+
+export function TvlBreakdownPageHeader({
+  title,
+  slug,
+  tvlBreakdownDate,
+}: Props) {
+  return (
+    <div className="mt-11 flex flex-col px-4 md:px-0">
+      <div className={'flex items-center gap-1.5'}>
+        <Image
+          width={16}
+          height={16}
+          src={`/icons/${slug}.png`}
+          alt={`${title} logo`}
+        />
```

Comment:
not a work for you but we should probably create component `ProjectLogo` with only slug passed to it and different size variants